.github/workflows/test.yml

```diff
@@ -20,6 +20,39 @@ jobs:
     uses: salesforcecli/github-workflows/.github/workflows/tarballs.yml@main
     secrets: inherit
 
+  manifest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          cache: yarn
+      - run: yarn install
+      - run: yarn build
+      - run: yarn oclif manifest
+      - uses: actions/upload-artifact@v2
+        with:
+          name: manifest
+          path: oclif.manifest.json
+
+  jit-install:
+    needs: [manifest]
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
Wouldn't this cause the JIT installs to run twice for linux? They will have already ran in the `tarballs` job above ([source](https://github.com/salesforcecli/github-workflows/blob/d92532a5b5f76b77de2e080d2c51da8fa4a58859/.github/workflows/tarballs.yml#L50C10-L50C10))

Reply:
@iowillhoit Yeah I wasn't sure what to do about that.

The existing test will run the smoke tests (which include JIT tests) on the tarballs produced by `oclif pack` whereas these new tests run the JIT tests on the local `bin/run`. I felt weird about removing the JIT tests from the smoke tests and I felt weird about only running these new tests only on windows

I'll happily defer to your judgement here though

Reply:
Ah that is a good distinction. I am fine leaving this testing both. Honestly linux will finish faster than windows anyway so it wont be adding any more time.